laravel/app/Models/EvaluateData.php

```diff
@@ -18,5 +18,18 @@ class EvaluateData extends Model
      *
      * @var bool
      */
-    public $timestamps = false;
+    public $timestamps = true;
```

Comment:
The default is `true`, which means we can get rid of this line entirely. See [here](https://laravel.com/docs/8.x/eloquent#timestamps)